schema/indexer/config.go

```diff
@@ -0,0 +1,52 @@
+package indexer
+
+// Config species the configuration passed to an indexer initialization function.
+// It includes both common configuration options related to include or excluding
+// parts of the data stream as well as indexer specific options under the config
+// subsection.
+//
+// NOTE: it is an error for an indexer to change its common options, such as adding
+// or removing indexed modules, after the indexer has been initialized because this
+// could result in an inconsistent state.
+type Config struct {
```

Comment:
will this be set in the app.toml or?

Reply:
Yes the idea is that this is a section in app.toml, probably under the [indexer] section

Reply:
Can we then add the corresponding `toml` struct tag then? (Like done in other server/v2 config struct)

Reply:
Which tags do we want? `toml`, `mapstructure`, `json`? One of these? All three?

Reply:
toml, mapstructure and comment: https://github.com/cosmos/cosmos-sdk/blob/main/server/v2/api/grpc/config.go

Reply:
So no json needed?

Reply:
I went ahead and added toml, mapstructure and comment and also kept JSON: https://github.com/cosmos/cosmos-sdk/pull/21636/commits/c967f789c051586488de27e18a1d8f6a0ee8c584. Does that look good?